apps/server/src/server.module.ts

```diff
@@ -59,6 +60,8 @@ const userEntities = [User, Role, Account];
 			},
 			// debug: true, // use it for locally debugging of querys
 		}),
+		// TODO use config or argv here?
+		MongoConsoleModule.forRoot({ publish: Configuration.get('PUBLISH_MONGO_CONSOLE') as boolean }),
```

Comment:
how to flag? configuration, start with specific argv given, use controller method decorators instead of module for root?

Reply:
maybe we should implement `ServerModule.forRoot()` so that we can configure, which submodultes are imported. that would also be useful when we want to use another database module in e2e tests or maybe for rabbitmq in dev environments.